db/view/view_update_generator.cc

```diff
@@ -39,10 +39,25 @@ future<> view_update_generator::start() {
                 if (result == stop_iteration::yes) {
                     break;
                 }
-                entry.t->move_sstable_from_staging_in_thread(entry.sst);
+                // collect all staging sstables to move in a map, grouped by table.
+                auto t = entry.t.get();
+                auto it = _sstables_to_move.find(t);
+                if (it == _sstables_to_move.end()) {
+                    _sstables_to_move.emplace(t, entry);
+                } else {
+                    it->second.sstables.push_back(entry.sst);
+                }
                 _registration_sem.signal();
                 _sstables_with_tables.pop_front();
             }
+            // For each table, move the processed staging sstables into the table's base dir.
+            for (auto it = _sstables_to_move.begin(); it != _sstables_to_move.end(); ) {
+                auto& entry = it->second;
+                try {
+                    entry.t->move_sstables_from_subdirs(entry.sstables).get();
+                } catch (...) { /* ignore */ }
```

Comment:
Why are we ignoring this?

Reply:
1. If moving sstables from the staging directory fails, they will be retried on restart.
2. Individual errors are already logged by move_sstables_from_subdirs so no need for another log message here.

I'll add that to a comment...